templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### What does this PR do?
+ 
+* 
+
+#### Merge Checklist:
+
+- [ ] The branch is up-to-date (i.e. rebased) with the base branch
+- [ ] All tests are passing - see ["Running tests"](../#running-tests)
+- [ ] No code style issues - see ["Checking code for potential issues"](../#checking-code-for-potential-issues)
```

Comment:
Do we have these sections? ("running tests & checking code for potential issues"?)

Reply:
Yes, they are on the README_TEMPLATE, forgot to update that file on the table of contents, I'll change it!